### PR TITLE
fix(element-templates): display proper error label

### LINKED ISSUE #166: Default error rendering: no label for error
__Describe the Bug__

Default error rendering: no label for error

__Steps to Reproduce__

Screenshot_20211109_195111

1. Apply this template:
2.
```{
    "$schema": "https://unpkg.com/@camunda/element-templates-json-schema@0.4.0/resources/schema.json",
    "name": "Default Errors Rendering",
    "id": "com.camunda.example.DefaultErrorsRendering",
    "appliesTo": [
      "bpmn:ServiceTask"
    ],
    "properties": [
      {
        "type": "Hidden",
        "value": "external",
        "binding": {
          "type": "property",
          "name": "camunda:type"
        }
      },
      {
        "value": "${true}",
        "binding": {
          "type": "camunda:errorEventDefinition",
          "errorRef": "error-1"
        }
      }
    ],
    "scopes": [
      {
        "type": "bpmn:Error",
        "id": "error-1",
        "properties": [
          {
            "type": "Hidden",
            "value": "500",
            "binding": {
              "type": "property",
              "name": "errorCode"
            }
          },
          {
            "type": "Hidden",
            "value": "an example error",
            "binding": {
              "type": "property",
              "name": "camunda:errorMessage"
            }
          },
          {
            "type": "Hidden",
            "value": "myError",
            "binding": {
              "type": "property",
              "name": "name"
            }
          }
        ]
      }
    ]
  }
```

3. Notice that the error will always have the label `empty`

__Expected Behavior__

Label should reflect the error config (compare existing implementation)
Screenshot_20211109_195347

__Environment__

https://github.com/bpmn-io/bpmn-properties-panel/pull/161

### PR DESCRIPTION
**Pull request by barmac**
Closes #166

### CODE REVIEW

**Comment by barmac** [comment]
Ready for review.

File: test/spec/provider/camunda-platform/Error.spec.js [hunk 1]
```diff
@@ -216,9 +216,11 @@ describe('provider/camunda-platform - Error', function() {
       const select = getErrorRefSelect(container, 'ServiceTask_Error');
 
       // then
+      console.log(asOptionNamesList(select));
```
Comment (MaxTru) [inline test/spec/provider/camunda-platform/Error.spec.js:219]:
```suggestion
```

File: test/spec/provider/element-templates/properties/ErrorProperties.spec.js [hunk 2]
```diff
@@ -46,6 +49,34 @@ describe('provider/element-templates - ErrorProperties', function() {
   }));
 
 
+  it('should display given label', async function() {
```
Comment (MaxTru) [inline test/spec/provider/element-templates/properties/ErrorProperties.spec.js:52]: Should we not add a testCase to `should show placeholder label if only code is set`? (analog to the plain error implementation

**Review by MaxTru — CHANGES_REQUESTED** [review]
Generally works, only minor comments

File: test/spec/provider/camunda-platform/Error.spec.js (hunk 1 above)
Comment (barmac) [reply test/spec/provider/camunda-platform/Error.spec.js:219]: 🙈

Comment (barmac) [reply test/spec/provider/camunda-platform/Error.spec.js:219]: BTW this is a thing which I'd expect to be caught by the linter 🗡️

Comment (MaxTru) [reply test/spec/provider/camunda-platform/Error.spec.js:219]: True

File: test/spec/provider/element-templates/properties/ErrorProperties.spec.js (hunk 2 above)
Comment (barmac) [reply test/spec/provider/element-templates/properties/ErrorProperties.spec.js:52]: I'll add it.

**Comment by barmac** [comment]
Ready for re-review.

**Comment by nikku** [comment]
General question:

* Is there a way we can try that one out in the test suite? I'd assume `npm run start:platform` makes templates available to me, but it does not.

**Comment by nikku** [comment]
Does not match existing behavior at the moment. If we want to improve beyond properties-panel@current we could display <unnamed> (code=foo) for errors without a name, but a code. 

Empty feels odd in this case, as the code is there:

**Comment by nikku** [comment]
Cf. attached screens from the old implementation: https://github.com/bpmn-io/bpmn-properties-panel/issues/166#issuecomment-970293459

**Comment by nikku** [comment]
Looks fine implementation wise :+1:

**Comment by barmac** [comment]
> General question:
> 
> * Is there a way we can try that one out in the test suite? I'd assume `npm run start:platform` makes templates available to me, but it does not.

`npm run start:templates`

**Comment by barmac** [comment]
Let's display then:

| error name | error code | label |
| ----------- | ----------- | ----| 
| name | 500 | name (code=500) |
| name | | name |
| | 500 | \<unnamed> (code=500) |
| | | \<unnamed> |

**Comment by barmac** [comment]
@nikku Implemented. I will also apply it to changes in https://github.com/bpmn-io/bpmn-properties-panel/pull/161

**Review by nikku — APPROVED** [review]

**Pull request merged**